faasmcli/faasmcli/tasks/libs.py

```diff
@@ -231,57 +218,43 @@ def eigen(ctx, verbose=False):
     ]
     cmd_string = " ".join(cmd)
 
-    res = call(cmd_string, shell=True, cwd=build_dir)
-    if res != 0:
-        exit(1)
+    run(cmd_string, shell=True, cwd=build_dir, check=True)
 
-    res = call(
-        "{} make install".format(verbose_string), shell=True, cwd=build_dir
+    run(
+        "{} make install".format(verbose_string),
+        shell=True,
+        cwd=build_dir,
+        check=True,
     )
-    if res != 0:
-        exit(1)
 
 
 @task
-def png(ctx):
-    """
-    Compile and install libpng
-    """
-    workdir = join(PROJ_ROOT, "third-party", "libpng")
-
-    config_cmd = BASE_CONFIG_CMD
-    config_cmd.extend(BASE_CONFIG_FLAGS)
-    config_cmd.extend(
-        [
-            "./configure",
-            "--build={}".format(WASM_BUILD),
-            "--host={}".format(WASM_HOST),
-            "--prefix={}".format(WASM_SYSROOT),
-        ]
+def clapack(ctx, clean=False):
+    # See the CLAPACK docs: http://www.netlib.org/clapack/
+    proj_dir = join(THIRD_PARTY_DIR, "faasm-clapack")
+    clapack_dir = join(proj_dir, "third-party", "clapack")
+    lapacke_dir = join(proj_dir, "third-party", "lapack", "LAPACKE")
+
+    # Build clapack
+    if clean:
+        run("make clean", cwd=clapack_dir, shell=True, check=True)
+        run("make clean", cwd=lapacke_dir, shell=True, check=True)
+
+    n_cpu = int(cpu_count()) - 1
+
+    # Make libf2c first (needed by others)
+    run(
+        "make f2clib -j {}".format(n_cpu),
+        shell=True,
+        cwd=clapack_dir,
+        check=True,
```

Comment:
This is new. `zlib` and `libpng` were removed a while ago but I missed these tasks